### PR TITLE
More fft examples

### PR DESCRIPTION
**Pull request by rhooper**
Adds 3 more examples that use the STEMMA port:
- v2 - Colour per row (like the original), that turns the whole bar red on peak.  Uses fractional brightness on the top of each column.
- v3 - Like v2, but only turns the top cell red on peak.
- v4 - Like v2/v3 but the colour for all pixels in a column are the same (brightness of top cell varies).

### CODE REVIEW

**Comment by ladyada** [comment]
could you combine this into one example with different modes? it sounds like they're quite similar

**Comment by rhooper** [comment]
Sure thing, that was my first inclination, but I wanted feedback first!

**Comment by rhooper** [comment]
Okay, I've collapsed most of the features into the original example and made them customizable.  I did use #ifdefs for performance.  I also moved the `trellis.show()`  outside the loop which seems to work fine.

File: examples/trellis_audio_fft/trellis_audio_fft.ino [hunk 1]
```diff
@@ -1,21 +1,54 @@
...
+// Whether to vary the brightness for partial values at the top of a bar
+#define VARY_BAR_TOP_BRIGHTNESS 1
+// Change to 0 to disbale printing the FFT to serial, 1 to enable.
+#define PRINT_FFT 0
+// Pause between loops.  Good values seem to be between 5 and 15 ms.  Low values may appear to flicker too much)
+#define DELAY 5
+
+// Choose an audio input
+//AudioInputAnalogStereo  audioInput(PIN_MIC, 0);
+AudioInputAnalogStereo  audioInput(PIN_LINE_LEFT, PIN_LINE_RIGHT);
```
Comment (ladyada) [inline examples/trellis_audio_fft/trellis_audio_fft.ino]: keep the default behaviour the same as before please!

**Pull request merged**